packages/feedback/src/lib/FeedbackForm.tsx

```diff
@@ -0,0 +1,159 @@
+interface SmileButtonProps extends ToggleButtonProps {
+  disabled: boolean;
+  Icon: (props: SvgIconProps) => JSX.Element;
+  label: string;
+  value: string;
+}
+
+interface FeedbackFormProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  analytics: { info: (entries: any) => any };
+  appName: string;
+  handleClose: () => void;
+  loading: boolean;
+  sent: boolean;
+  setLoading: React.Dispatch<React.SetStateAction<boolean>>;
+  setSent: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const SmileButtons = styled(ToggleButtonGroup, { name: 'AvFeedbackContainer', slot: 'SmileButtons' })({});
+
+const SmileButton = ({ disabled, Icon, label, value, ...props }: SmileButtonProps) => (
+  <div>
+    <ToggleButton title={label} value={value} {...props} disabled={disabled}>
+      <Icon fontSize="large" />
+    </ToggleButton>
+  </div>
+);
+
+export const FeedbackForm = ({
+  analytics,
+  appName,
+  handleClose,
+  loading,
+  sent,
+  setLoading,
+  setSent,
+}: FeedbackFormProps): JSX.Element | null => {
+  const { control, handleSubmit, setValue, watch } = useForm<Inputs>();
+
+  const onSubmit: SubmitHandler<Inputs> = async ({ smileField, ...values }) => {
+    setLoading(true);
+    try {
+      const response = await avRegionsApi.getCurrentRegion();
+
+      await analytics.info({
+        surveyId: `${appName.replace(/\s/g, '_')}_Smile_Survey`,
+        smileLocation: `${appName}`,
+        smile: `icon-${smileField}`,
+        url: window.location.href,
+        region: response.data.regions[0] && response.data.regions[0].id,
+        userAgent: window.navigator.userAgent,
+        submitTime: new Date(),
+        ...values, // Spread the form values onto the logger
+      });
+      setSent(true);
+      setLoading(false);
+    } catch {
+      setSent(false);
+      setLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    if (sent) {
+      setTimeout(() => {
+        handleClose();
+      }, 2000);
+    }
+  }, [sent]);
+
+  const options = [
+    {
+      Icon: FaceSmileIcon,
+      label: 'What do you like?',
+      value: 'smile',
+    },
+    {
+      Icon: FaceNeutralIcon,
+      label: 'What would you improve?',
+      value: 'meh',
+    },
+    { Icon: FaceFrownIcon, label: "What don't you like?", value: 'frown' },
+  ];
+  if (!sent) {
+    return (
+      <Grid component="form" container justifyContent="center" onSubmit={handleSubmit(onSubmit)}>
+        <Controller
+          control={control}
+          name="smileField"
+          render={({ field }) => {
+            return (
+              <SmileButtons
+                children={options.map((props) => (
+                  <SmileButton disabled={loading} key={props.value} {...props} />
+                ))}
+                {...field}
+                onChange={(event: React.MouseEvent<HTMLElement>, value: string) => {
+                  setValue(field.name, value);
+                }}
+                size="medium"
+                exclusive
```

Comment:
I believe we previously used the title as the label for the button group.

Reply:
done